bender.mk

```diff
@@ -8,4 +8,4 @@
 COMMON_TARGS ?=
 COMMON_TARGS += -t snitch_cluster -t cv32a6_convolve -t cva6 -t rtl
 
-SIM_TARGS = -t test -t sim
+SIM_TARGS = $(COMMON_TARGS) -t test -t sim
```

Comment:
Why do you add here $(COMMON_TARGS)? Isn't the variable already used in `sim.mk`?

Reply:
[This also answers https://github.com/pulp-platform/chimera/pull/47#discussion_r1808717677]
Just a small modification to make (in my opinion) the flow clearer. At some point we'll have many targets in the `bender.mk`, e.g. `SIM_TARGS`, `ASIC_TARGS`, `FPGA_XILING_TARGS`, `FPGA_whatever_TARGS`, etc...
So I explicitly add `$(COMMON_TARGS)` (which is common to all of them) within all those other TARGS variables.
Previously, it was left to whoever wrote the independent target `.mk` (e.g., `sim.mk`, `asic.mk`, ...) to use both variables, as in: `bender script vsim $(COMMON_TARGS) $(SIM_TARGS) [...]`.
But ultimately it's the same argument for `bender` so we might as well use this clearer form.

Reply:
make sense, thanks